.github/workflows/main.yml

```diff
@@ -43,16 +67,23 @@ jobs:
         - "pre-quincy"
         - "pre-reef"
         - "main"
+        go_version:
+        - ${{ needs.go-versions.outputs.latest }}
+        include:
+        - ceph_version: "reef"
+          go_version: ${{ needs.go-versions.outputs.prev }}
+        - ceph_version: "reef"
+          go_version: ${{ needs.go-versions.outputs.unstable }}
```

Comment:
Should we include `main` variant along with `reef` to run against multiple Go versions?

Reply:
I thought "latest stable" is better, because I expect main to break more easily, and then we lose all go version. Or you mean additionally? We have plenty jobs already, and we should not forget they cost energy (although we don't pay it directly). What additional information do you hope to get?

Reply:
Yes, I meant an additional entry for `main` using previous and unstable go versions. Our intention to detect/tackle ongoing changes(from Ceph upstream) with current `main` job can be extended to run against upcoming changes with golang too.

Reply:
But it's the same go-ceph code. I don't see the direct touch-points between ceph code and Go version. I don't think you would increase coverage. Do you have an example in mind, that would fail on main with the latest Go version but not on reef with the latest Go version? (It could happen if we have go-ceph code that is only compiled on main, but I don't think we have that.)

Reply:
How we merge it as is, and then we can add main coverage in another PR, if we think that makes sense.

Reply:
Fine by me.